llama-index-integrations/embeddings/llama-index-embeddings-cohere/llama_index/embeddings/cohere/base.py

```diff
@@ -94,7 +105,10 @@ class CohereEmbedding(BaseEmbedding):
     """CohereEmbedding uses the Cohere API to generate embeddings for text."""
 
     # Instance variables initialized via Pydantic's mechanism
-    cohere_client: Any = Field(description="CohereAI client")
+    _cohere_client: cohere.Client = Field(description="CohereAI client")
```

Comment:
You may run into some issues here with Pydantic if you don't set this as a PrivateAttr instead of Field. You can import PrivateAttr from our bridge as you do with Field

Reply:
Looks like these will get converted to PrivateAttr by Pydantic. It shouldn't be an issue since your defining them at construction.

Reply:
@nerdai sorry i made a mistake with testing and realised that the _ broke the class. Please could you accept the revert or suggest alternative? thanks! https://github.com/run-llama/llama_index/pull/12746